src/components/DatePicker/DatePicker.scss

```diff
@@ -11,6 +11,23 @@ $font-size: rem(12px);
 // stylelint-disable-next-line color-no-hex
 $in-range-border-color: #9ca6de;
 
+@mixin left-half-circle {
+  border-radius: 50% 0 0 50%;
+}
+
+@mixin right-half-circle {
+  border-radius: 0 50% 50% 0;
+}
+
+@mixin radius-with-focus-ring {
+  @content;
+  &:focus:not(:active) {
+    &::after {
+      @content;
+    }
+  }
+}
+
```

Comment:
```suggestion
```

Reply:
These are only used once or twice and with my proposed changed below they won't be needed.